src/components/DropDown/index.tsx

```diff
@@ -0,0 +1,13 @@
+import styles from "./DropDown.module.scss";
+import { IDropDownProps } from "./types";
+const DropDown: React.FC<IDropDownProps> = ({ items }) => {
+  return (
+    <select className={styles["drop-down"]}>
+      {items.map((e, i) => {
+        return <option key={i}>{e}</option>;
+      })}
```

Comment:
refactor it to remove `return`

Reply:
https://github.com/igorizviekov/light-up/pull/7/commits/794e3f1b32505b24dd56d4254d2c216adbb5ee4d